.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You should not be checking this in because it will overwrite it for others and we have local settings in vscode that will be overwritten because of this.

Reply:
Agreed!